Screenbox/Strings/en-US/Resources.resw

```diff
@@ -803,20 +824,26 @@
   <data name="GetLivelyVisuals" xml:space="preserve">
     <value>Download visuals on GitHub</value>
   </data>
+  <data name="VisualizerOptionsToolTip" xml:space="preserve">
+    <value>Audio visual options</value>
+  </data>
+  <data name="Visualizers" xml:space="preserve">
+    <value>Visuals</value>
+  </data>
```

Comment:
Don't create a string called "Visualizers" and then set it to "Visuals". You should either be more descriptive with the string name or match the value with the string name. We may need to use the word "visualizers" in the future.

Reply:
Oops, it slipped through, I'll fix it